frankenstein/prog/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/prometheus/common/log"
+	"github.com/prometheus/prometheus/frankenstein"
+)
+
+func main() {
+	var (
+		listen       string
+		consulHost   string
+		consulPrefix string
+	)
+
+	flag.StringVar(&listen, "listen", ":80", "HTTP server listen address")
```

Comment:
A default port in the 9xxx range would be good, also to make it work without root privileges. Maybe 9094, since that's not used yet in https://github.com/prometheus/prometheus/wiki/Default-port-allocations? Or open a new range somewhere above the expected number of exporters (like 95xx?). Anyways, that can be decided later.